cmd/cli/create/create.go

```diff
@@ -197,7 +197,7 @@ func runProvision(log *logger.FunLogger, opts *options) error {
 
 	// Download kubeconfig
 	if opts.cfg.Spec.Kubernetes.Install && (opts.cfg.Spec.Kubernetes.KubeConfig != "" || opts.kubeconfig != "") {
-		if opts.cfg.Spec.Kubernetes.KubernetesInstaller == "microk8s" || opts.cfg.Spec.Kubernetes.KubernetesInstaller == "kind" {
+		if opts.cfg.Spec.Kubernetes.KubernetesInstaller == "microk8s" || opts.cfg.Spec.Kubernetes.KubernetesInstaller == "kind" || opts.cfg.Spec.Kubernetes.KubernetesInstaller == "nvkind" {
```

Comment:
Nit: This conditional is getting quite long. What about:
```suggestion
		doesNotSupportKubeconfigRetrieval := map[string]bool{
			"mikrok8s": true,
			"kind": true,
			"nvkind": true,
		}
		if doesNotSupportKubeconfigRetrieval[opts.cfg.Spec.Kubernetes.KubernetesInstaller] {
```

Reply:
adopted